components/ipfs_ui/BUILD.gn

```diff
@@ -1,17 +1,23 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
 import("//brave/components/common/typescript.gni")
 
+assert(!is_android && !is_ios)
+
 transpile_web_ui("ipfs_ui") {
-  entry_points = [
-    ["ipfs", rebase_path("ipfs.tsx")]
-  ]
+  entry_points = [ [
+        "ipfs",
+        rebase_path("ipfs.tsx"),
```

Comment:
nit : some unnecessary formatting in this file

Reply:
we use `gn format`, so some weird formatting is possible, but at least it is consistent.